src/main/kotlin/com/cognifide/gradle/aem/pkg/deploy/SyncTask.kt

```diff
@@ -11,15 +11,24 @@ abstract class SyncTask : AemDefaultTask() {
     protected val propertyParser = PropertyParser(project)
 
     protected fun synchronizeInstances(synchronizer: (InstanceSync) -> Unit) {
-        synchronizeInstances(synchronizer, filterInstances())
+        synchronizeInstances(synchronizer, Instance.filter(project))
     }
 
     protected fun <T : Instance> synchronizeInstances(synchronizer: (InstanceSync) -> Unit, instances: List<T>) {
-        val callback = { instance: T -> synchronizeInstance(synchronizer, instance) }
-        if (config.deployParallel) {
-            instances.parallelStream().forEach(callback)
+        synchronizeInstances(instances, { instance: T -> synchronizeInstance(synchronizer, instance) })
+    }
+
+    protected fun <T : Instance> synchronizeInstances(instances: List<T>, callback: (T) -> Unit) {
+        if (instances.isEmpty()) {
+            logger.warn("No instances to synchronize. Verify deploy instance name filter '${config.deployInstanceName}' and defined instances name (environment-type).")
```

Comment:
clue of change, now plugin user will be informed why satisfy is not working if he define `remoteInstance('x.x.x.x:4502', 'user', 'password', 'int')`

he will need to update aem config or specify cmd property `-Paem.deploy.instance.name=int-*` to deploy packages by satisfy task (that parameter intentionally protects against deployments other to `local` env by default)